### PR TITLE
Add certified robustness tracking and post-transformation re-verification to C11.2

### LINKED ISSUE #720: C11.2 lacks concrete certification and measurement methods  inconsistent with C12.3's approach to Differential Privacy
## Problem

C12.3 (Differential-Privacy Safeguards) sets a strong precedent for how the AISVS handles formal guarantees: it requires tracking specific metrics per training round (ε and δ values), mandates empirical audits with stated confidence levels (Clopper-Pearson or f-DP confidence intervals), and extends certification coverage to post-training steps (fine-tunes, embedding generation). This gives auditors clear, measurable, and reproducible verification criteria.

C11.2 (Adversarial-Example Hardening) addresses an analogous problem, that provide formal guarantees about model robustness against adversarial inputs, however, does not apply the same rigor. The controls remain at an abstract level ("evaluate against known techniques," "apply adversarial training," "use adaptive attacks") without specifying **what to measure, how to measure it, or how to ensure measurements remain valid across the model lifecycle.**

The only control that references formal methods is 11.2.5 ("formal robustness verification methods, e.g., certified bounds, interval-bound propagation"), but it is Level 3 and stands alone, i.e. there is no supporting structure for tracking, auditing, or maintaining these guarantees over time, unlike the three-tier structure in C12.3.

## Why This Matters

- **Auditability gap**: An auditor assessing C12.3 compliance knows exactly what evidence to request (ε consumption logs, empirical ε̂ estimates with confidence bounds, formal proof coverage). An auditor assessing C11.2 has no equivalent criteria  "evaluated against known adversarial attack techniques" (11.2.1) does not specify which metrics to report, at what thresholds, or with what statistical confidence.
- **Post-training blind spot**: Quantization, pruning, distillation, LoRA merging, and other post-training transformations can silently degrade certified robustness guarantees, just as they can degrade privacy guarantees. C12.3.3 explicitly addresses this for privacy. C11.2 has no equivalent, meaning a model could be certified robust at training time and deployed in a weakened state after transformation.
- **Methodological maturity**: The certified robustness field has well-established, auditable metrics certified radius, verified robust accuracy at defined perturbation budgets (e.g., ℓ∞ ε-balls), empirical robustness benchmarks like AutoAttack with standardized parameters. These are as mature and measurable as the differential privacy metrics referenced in C12.3, yet the standard does not leverage them.

## Proposed Changes

Add two requirements to C11.2 that mirror the measurement and lifecycle structure of C12.3:

| # | Description | Level |
|---|---|---|
| 11.2.6 | **Verify that** certified robustness metrics (e.g., certified radius, verified robust accuracy at defined perturbation bounds) are tracked and recorded per model version, and that degradation beyond defined thresholds triggers re-evaluation before deployment. | 2 |
| 11.2.7 | **Verify that** robustness certification or empirical robustness audits are repeated after all post-training transformations (fine-tuning, distillation, quantization, adapter merging) that consume the same base model. | 3 |

### Rationale for the parallel structure

| Aspect | C12.3 (Privacy)  existing | C11.2 (Robustness)  proposed |
|---|---|---|
| **Per-version metric tracking** | 12.3.1: Track ε and δ per training round, alert on threshold breach | 11.2.6: Track certified radius / robust accuracy per model version, alert on threshold breach |
| **Empirical audits with confidence** | 12.3.2: Black-box privacy audits estimating ε̂ with Clopper-Pearson bounds | Covered implicitly by 11.2.4 (adaptive attacks), but could benefit from explicit confidence-level language in a future iteration |
| **Post-training coverage** | 12.3.3: Formal proofs cover post-training fine-tunes and embedding steps | 11.2.7: Robustness certification repeated after post-training transformations |

## Scope

> **Scope note for 11.2.7:** C3.2.7 requires re-evaluation of quantized/pruned/distilled models against the same safety and alignment test suite, and C3.2.4 may imply inclusion of adversarial robustness in that suite. However, 11.2.7 addresses a distinct concern: formal or empirical **robustness certification** (certified radius, verified robust accuracy at defined perturbation bounds) is not equivalent to passing a general safety/alignment test suite. A model can pass behavioral safety tests while losing certified robustness guarantees after transformation. 11.2.7 explicitly requires that the robustness metrics tracked under 11.2.6 are re-verified, not merely that the model passes a functional test suite. Auditors should treat C3.2.7 and 11.2.7 as complementary, i.e. the former covers behavioral safety regression, the latter covers formal robustness guarantee preservation.

This proposal only adds requirements , i.e.  it does not modify existing 11.2.1–11.2.5 controls. It brings C11.2 into structural alignment with C12.3 so that both formal-guarantee sections of the standard follow the same measure → audit → maintain-across-lifecycle pattern.

Relates to: C11.2 (Adversarial-Example Hardening), C12.3 (Differential-Privacy Safeguards), C03.x (Model Lifecycle Management)

### PR DESCRIPTION
**Pull request by BarnoKa**
## Summary

Fixes #720 

Adds two requirements to C11.2 (Adversarial-Example Hardening) that bring it into structural alignment with C12.3 (Differential-Privacy Safeguards). C12.3 requires per-version metric tracking (12.3.1), empirical audits with confidence bounds (12.3.2), and post-transformation coverage (12.3.3). C11.2 currently has none of these despite certified robustness metrics being equally mature and auditable.

Includes a scope note distinguishing 11.2.7 from C3.2.7 to prevent auditor confusion.

## Motivation

C12.3 gives auditors clear, measurable criteria: ε/δ consumption logs, empirical ε̂ estimates with Clopper-Pearson bounds, and formal proof coverage across post-training steps. C11.2 stays abstract,  "evaluate against known techniques," "apply adversarial training", i.e. without specifying what to measure or how to ensure measurements survive the model lifecycle.

The certified robustness field has equivalent tooling and metrics: certified radius, verified robust accuracy at defined perturbation budgets (ℓ∞ ε-balls), and standardized benchmarks (AutoAttack). These are direct analogues to the DP metrics in C12.3.

## Changes

- **File:** `1.0/en/0x10-C11-Adversarial-Robustness.md`
- **Section:** C11.2 Adversarial-Example Hardening
- **Change:** Add two requirement rows after 11.2.5, plus a scope note for 11.2.7
- **Scope:** Two additions, no existing controls modified

## Requirements added

| # | Description | Level |
|---|---|---|
| 11.2.6 | **Verify that** certified robustness metrics (e.g., certified radius, verified robust accuracy at defined perturbation bounds) are tracked and recorded per model version, and that degradation beyond defined thresholds triggers re-evaluation before deployment. | 2 |
| 11.2.7 | **Verify that** robustness certification or empirical robustness audits are repeated after all post-training transformations (fine-tuning, distillation, quantization, adapter merging) that consume the same base model. | 3 |

## Scope note (included in file change)

11.2.7 is complementary to C3.2.7, not redundant. C3.2.7 requires re-running the safety/alignment test suite after transformation, i.e. a behavioral regression check. 11.2.7 requires re-verifying formal robustness certification, a guarantee preservation check. A model can pass one and fail the other.

## Parallel structure with C12.3

| Aspect | C12.3 (Privacy) existing | C11.2 (Robustness) proposed |
|---|---|---|
| Per-version metric tracking | 12.3.1: Track ε and δ per training round | 11.2.6: Track certified radius / robust accuracy per model version |
| Post-training coverage | 12.3.3: Formal proofs cover post-training steps | 11.2.7: Robustness certification repeated after transformations |

### CODE REVIEW

**Review by RicoKomenda — CHANGES_REQUESTED** [review]
Good motivation and the parallel with C12.3 is well-made. Two structural issues need fixing before this can merge.

## 1. Level-ordering violation (blocking)

The new 11.2.6 is L2 but is placed after two L3 controls (11.2.4 and 11.2.5). Every section in AISVS must read L1→L2→L3 top to bottom (enforced across the full series in #705). As written, an auditor reading the table would encounter:

```
11.2.3  L2
11.2.4  L3  ← existing
11.2.5  L3  ← existing
11.2.6  L2  ← new — out of order
11.2.7  L3  ← new
```

The fix is to insert 11.2.6 before 11.2.4, which requires renumbering the existing L3 controls:

| New # | Old # | Level | Description |
|---|---|---|---|
| 11.2.4 | _(new)_ | 2 | certified robustness metrics tracked per version |
| 11.2.5 | 11.2.4 | 3 | adaptive attack evaluations |
| 11.2.6 | 11.2.5 | 3 | formal robustness verification |
| 11.2.7 | _(new)_ | 3 | post-transformation re-certification |

Appendix D cross-references for 11.2.4 and 11.2.5 would need updating too (both are referenced in AD.15).

## 2. Scope note placement (blocking)

The scope note for 11.2.7 is placed in the chapter-level preamble — between the existing chapter scope note and the first  divider — before any section has been introduced. A reader encountering it there would not yet know what 11.2.7 is. Every other section-specific scope note in the document sits inside or directly above the relevant section table (e.g. the scope notes in C10.2, C10.5, C10.6, C11.8, C14.2).

Move the scope note into the C11.2 section, either as a blockquote directly above or below the table.

## Minor

The scope note itself is correct and clear — the distinction between behavioral safety regression (C3.2.7) and formal robustness guarantee preservation (11.2.7) is meaningful and worth keeping. It just needs to live in the right place.

**Review by RicoKomenda — APPROVED** [review]
Pushed a fixup commit directly to the branch (maintainer edit). Both issues are resolved:
- Scope note moved into the C11.2 section, above the table.
- Level ordering fixed: 11.2.4 is now the new L2 (certified robustness tracking), existing L3 controls shifted to 11.2.5 and 11.2.6, new L3 post-transformation control stays at 11.2.7.
- Appendix D updated: 11.2.5 ref corrected to 11.2.6; entries added for 11.2.4 and 11.2.7.

Good to merge from my side.
